src/expressions/dataTypes/casting/tryCastToType.ts

```diff
@@ -159,11 +157,12 @@ function createCastingFunction(from: ValueType, to: ValueType) {
 		});
 	}
 
-	const converters = [];
+	const converters: ((value: AtomicValue) => CastResult)[] = [];
 
 	if (primitiveFrom === ValueType.XSSTRING || primitiveFrom === ValueType.XSUNTYPEDATOMIC) {
 		// We are dealing with string-like types. Check whitespace / pattern
-		converters.push((value) => {
+		// TODO: This code seems broken!
```

Comment:
I want to make a separate issue to see why this code is not working as it should: the converters should accept Value and return Value, but _some_ of the converters are working with strings and should crash when passed Value. Which is strange. Likely this code is just dead.